pkg/app/app.go

```diff
@@ -66,13 +67,16 @@ func CreateApp() App {
 		app.log.Error(err.Error())
 		os.Exit(1)
 	}
-	serv, clust, err := app.dao.GetEtcdVersion(app.config.Dao)
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	defer cancelFunc()
+
+	version, err := app.dao.Client.GetVersion(ctx)
```

Comment:
This is going to end up using the new clients patch #222.  I'll wait for that to merge so I can avoid exposing dao.